### PR TITLE
Increased password max length

### PR DESCRIPTION
**Pull request by dugite-code**
Increased _password max_length from 100 to 255

As per #17

### CODE REVIEW

File: modoboa_imap_migration/migrations/0001_initial.py [hunk 1]
```diff
@@ -15,7 +15,7 @@ class Migration(migrations.Migration):
             name='Migration',
             fields=[
                 ('id', models.AutoField(verbose_name='ID', serialize=False, auto_created=True, primary_key=True)),
-                ('_password', models.CharField(max_length=100)),
+                ('_password', models.CharField(max_length=255)),
```
Comment (tonioo) [inline modoboa_imap_migration/migrations/0001_initial.py]: Don't do that, add a new migration instead. Look at https://docs.djangoproject.com/en/1.10/ref/django-admin/#django-admin-makemigrations.

Comment (tonioo) [reply modoboa_imap_migration/migrations/0001_initial.py]: Please ;-)

Comment (dugite-code) [reply modoboa_imap_migration/migrations/0001_initial.py]: Ah yes, quite right. I seem to have gotten myself turned around somewhere in my reading

**Comment by tonioo** [comment]
@dugite-code Could you also add a migration please?

**Comment by tonioo** [comment]
Thank you!

**Pull request merged**